### DISCUSSION TITLE
Keybinding not working fully

### DISCUSSION THREAD

**Discussion by Fred-IV** [MonitorControl/MonitorControl#995] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

My volume keys are working in Mac OS using the fn keys F9-F11.
The brightness keys in this app cannot be set for the fn keys.
This is interfering with games and other apps.
I've tried all the settings, and it seems to ignore the fn keys completely.

### Steps to reproduce

I've tried re-binding the keys in the settings, but it cannot recognize the fn key.

### Expected behavior

Bind keys using the fn key, for example fn F14-F15

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.1
- Mac model: Mac Pro 5,1
- MonitorControl version: Version 4.0.2 Build 6965
- Monitor(s): ViewSonic XG2705 27-inch (1920 × 1080)
- Apple Silicon/M1 (yes or no): no

Chipset Model:	Radeon RX 580
  Type:	GPU
  Bus:	PCIe
  Slot:	Slot-1
  PCIe Lane Width:	x16
  VRAM (Total):	8 GB
  Vendor:	AMD (0x1002)
  Device ID:	0x67df
  Revision ID:	0x00e7
  Metal Family:	Supported, Metal GPUFamily macOS 2
  Displays:
XG2705:
  Resolution:	1920 x 1080 (1080p FHD - Full High Definition)
  UI Looks like:	1920 x 1080 @ 60.00Hz
  Framebuffer Depth:	30-Bit Color (ARGB2101010)
  Display Serial Number:	
  Main Display:	Yes
  Mirror:	Off
  Online:	Yes
  Rotation:	Supported
  Automatically Adjust Brightness:	Yes
  Connection Type:	Thunderbolt/DisplayPort


Gaming KB :

  Product ID:	0x002a
  Vendor ID:	0x258a
  Version:	16.63
  Speed:	Up to 12 Mb/s
  Manufacturer:	SINO WEALTH
  Location ID:	0xfa240000 / 2
  Current Available (mA):	500
  Current Required (mA):	500
  Extra Operating Current (mA):	0
```

**Comment by waydabber** [comment]
Are you trying to create custom keyboard shortcuts? If so and it does not recognize these keys, you can file a request here:

https://github.com/sindresorhus/KeyboardShortcuts#localization

MonitorControl uses this opensource tool to handle custom keyboard shortcuts.

Btw F14/F15 should be handled by the Standard (non custom) media key settings. There is a good chance that switching to that will fix the issue:

<img width="842" alt="Screen Shot 2022-04-07 at 19 58 59" src="https://user-images.githubusercontent.com/37590873/162267139-964ce498-6084-49aa-9b58-5d42d5560351.png">

**Reply by Fred-IV** [reply]
That screenshot is how mine is set now.
I have also tried "custom" and "both". Neither will work.
It just doesn't want to bind using the function key.
As far as what the default OS keys would be, I have no idea, because dim/bright never worked before. This app at least brings that functionality back.
My default OS keys for the volume use fn-F9-F11 (mute, vol down, vol up), so I know the OS is seeing the fn key properly.
I think it's just a matter of getting this app to see the fn key.

**Reply by StackSurfer** [reply]
Same Problem here. Is there any solution available now?
